app/config.pri.default

```diff
@@ -43,6 +43,9 @@ unix:!macx:!android {
 
   PROJ_INCLUDE_DIR = $${BASE_DIR}/include
   PROJ_LIB_DIR = $${BASE_DIR}/lib
+
+  # To use OGR API
+  OGR_INCLUDE_DIR = < path to gdal.h like /usr/include/gdal >
```

Comment:
where is gdal.h used?

Reply:
In ProjectWizard we use `QgsVectorFileWriter` and it is its dependency 
https://github.com/lutraconsulting/input/blob/master/app/projectwizard.cpp#L7

https://github.com/qgis/QGIS/blob/master/src/core/qgsvectorfilewriter.h#L32

Reply:
in that case it should be added to all platforms? (on iOS and Android it is in the same folder as proj, so it works probably, but on mac I am not sure @sklencar )?

Reply:
It has the same folder on MacOS (qgis-deps folder). I have not defined it explicitly since the path is included with proj variables.